iphone/Maps/Bookmarks/Categories/BMCViewModel/BMCDefaultViewModel.swift

```diff
@@ -117,8 +117,8 @@ extension BMCDefaultViewModel {
       return
     }
 
-    categories.append(manager.category(withId: manager.createCategory(withName: name)))
-    view?.insert(at: [IndexPath(row: categories.count - 1, section: section)])
+    categories.insert(manager.category(withId: manager.createCategory(withName: name)), at: 0)
```

Comment:
As I understood from your testing, there are some issues with this approach, right?

Let's remove these lines from this PR so it can be merged. And implement sorting in another PR.